### DISCUSSION TITLE
Add colors to tags

### DISCUSSION THREAD

**Discussion by davidevertuani** [mealie-recipes/mealie#4312] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I use tags to sort recipes by e.g. vegetarian, provided by friends, family recipes.... Tags are all colored in the same color; would love to be able to set a custom color to display the tag label below the recipe in the main UI.

### Please provide a concise description of the feature that would resolve your issue.

Add a color picker in the create/edit tag page; store the color and use it to display the tag in the UI.

### Please consider and list out some caveats or tradeoffs made in your design decision

No caveats.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
This has already been accepted and there is a open task for it, so we will get to it eventually.

See #1241
